README.md

```
@@ -1,92 +1,94 @@
+1. Most Recent Sleep Date
+    1. This is the most recent date for which data is available. This will match Current Data Date if you use your CPAP every day. An automation that triggers when these two sensors are different will signal that you have missed a night
+
+## Services
+
+Each config entry for this integration will create a service called `resmed_myair.force_poll_{username}` that will force an update from myAir.
+
+## Known Issues
+
+This integration was reversed engineered from the myAir website. There are no guarantees that this will continue to work, as this is up to the whims of ResMed. Please DO NOT rely on this for any health-related matters.
+
+This integration currently only connects to accounts from North America, Europe, and Australia. If you are in Asia and have access to the ResMed myAir website in your country (https://myair.resmed.com), please open an issue and offer yourself as a test subject.
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)